.github/workflows/tests.yml

```diff
@@ -17,18 +17,27 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
-    - uses: actions/setup-python@v2
+    - uses: actions/setup-python@v3
       with:
         python-version: 3.8
+        cache: 'pip'
```

Comment:
I think you have to [set the path](https://github.com/superdesk/superdesk-belga/blob/release/bel13/.github/workflows/tests.yml#L39) for cache to work